clang/test/BoundsSafety/CodeGen/auto-init/global-bidi-indexable-init-scalar.c

```diff
@@ -1,6 +1,6 @@
 
-// RUN: %clang_cc1 -S -fbounds-safety -O0 %s -o - | FileCheck %s
-// RUN: %clang_cc1 -S -fbounds-safety -x objective-c -fbounds-attributes-objc-experimental -O0 %s -o - | FileCheck %s
+// RUN: %clang_cc1 -S -fbounds-safety -triple arm64-apple-darwin -O0 %s -o - | FileCheck %s
```

Comment:
Nit: At some point we should try to be consistent with the target triples. Even better we should make this a lit substitution so at test time we can change it by passing `-Dbounds_safety_target_triple=x86_64-linux-unknown` to lit.